packages/core/src/components/tag/tag.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { newSpecPage } from '@stencil/core/testing'
+
+import { IconProps } from '../../icons'
+import { AtomTag } from './tag'
+
+const setup = async (color: AtomTag['type'], icon?: IconProps) => {
+  const currentIcon = icon && `icon=${icon}`
+
+  const page = await newSpecPage({
+    components: [AtomTag],
+    html: `<atom-tag type=${color} ${currentIcon}>Tag</atom-tag>`,
+  })
+
+  await page.waitForChanges()
+
+  return page.root?.shadowRoot
+}
+
+describe('atom-tag', () => {
+  it('should render atom-tag', async () => {
+    const tagLabel = 'Tag Test Label'
+    const page = await newSpecPage({
+      components: [AtomTag],
+      html: `<atom-tag type='success'>${tagLabel}</atom-tag>`,
+    })
+
+    await page.waitForChanges()
+
+    const atomTag = page.root?.nodeName.toLocaleLowerCase()
+    const atomTagLabel = page.root?.innerHTML
+    expect(atomTag).toBe('atom-tag')
+    expect(atomTagLabel).toBe(tagLabel)
+  })
+  it('should render atom-tag with icon', async () => {
+    const mockedIcon = 'heart'
+    const atomTag = await setup('success', mockedIcon)
+    expect(atomTag).toEqualHtml(`<ion-badge class="atom-tag" color="success">
+    <atom-icon class="atom-icon" icon=${mockedIcon}></atom-icon>
+    <slot></slot>
+  </ion-badge>`)
+  })
+  it('should render atom-tag success', async () => {
+    const mockedType: AtomTag['type'] = 'success'
+    const atomTag = await setup(mockedType)
+    expect(atomTag)
+      .toEqualHtml(`<ion-badge class="atom-tag" color=${mockedType}>
+    <slot></slot>
+  </ion-badge>`)
+  })
+  it('should render atom-tag danger', async () => {
+    const mockedType: AtomTag['type'] = 'danger'
+    const atomTag = await setup(mockedType)
+    expect(atomTag)
+      .toEqualHtml(`<ion-badge class="atom-tag" color=${mockedType}>
+    <slot></slot>
+  </ion-badge>`)
+  })
+  it('should render atom-tag warning', async () => {
+    const mockedType: AtomTag['type'] = 'warning'
+    const atomTag = await setup(mockedType)
+    expect(atomTag)
+      .toEqualHtml(`<ion-badge class="atom-tag" color=${mockedType}>
+    <slot></slot>
+  </ion-badge>`)
+  })
+  it('should render atom-tag dark', async () => {
+    const mockedType: AtomTag['type'] = 'dark'
+    const atomTag = await setup(mockedType)
+    expect(atomTag)
+      .toEqualHtml(`<ion-badge class="atom-tag" color=${mockedType}>
+    <slot></slot>
+  </ion-badge>`)
+  })
+  it('should render atom-tag light', async () => {
+    const mockedType: AtomTag['type'] = 'light'
+    const atomTag = await setup(mockedType)
+    expect(atomTag)
+      .toEqualHtml(`<ion-badge class="atom-tag" color=${mockedType}>
+    <slot></slot>
+  </ion-badge>`)
+  })
+  it('should render atom-tag neutral', async () => {
+    const mockedType: AtomTag['type'] = 'neutral'
+    const atomTag = await setup(mockedType)
+    expect(atomTag)
+      .toEqualHtml(`<ion-badge class="atom-tag" color=${mockedType}>
```

Comment:
(suggestion) to improve tests performance we could test just an arg about `color`, because we need just to test the render the value passed with params, what u think?

Reply:
@felipefialho i tried to assert only the `color` prop. See what you think https://github.com/juntossomosmais/atomium/pull/261/commits/8c06b3ce282f00724422af5d0d255291a0e30873

Reply:
@luiz-ortega maybe we could just test any color, do you know? we don't need to test one by one, because what we want to validate is the color being passed correctly

Reply:
@felipefialho now i inderstand what you meant: `test only one color` :sweat_smile:  Did it here https://github.com/juntossomosmais/atomium/pull/261/commits/543229fd2b85f225bfd31c37ccfcc8a6607ac03e